dipper/sources/GeneOntology.py

```diff
@@ -318,46 +387,46 @@ def process_gaf(self, file, limit, id_map=None, eco_map=None):
                 # snRNA; snoRNA; any subtype of ncRNA in the Sequence Ontology.
                 # If the precise product type is unknown,
                 # gene_product should be used
-                #######################################################################
+                ########################################################################
 
                 # Derive G2P Associations from IMP annotations
                 # in version 2.1 Pipe will indicate 'OR'
                 # and Comma will indicate 'AND'.
                 # in version 2.0, multiple values are separated by pipes
                 # where the pipe has been used to mean 'AND'
                 if eco_symbol == 'IMP' and with_or_from != '':
-                    withitems = re.split(r'\|', with_or_from)
-                    phenotypeid = go_id+'PHENOTYPE'
+                    withitems = with_or_from.split('|')
+                    phenotypeid = go_id + 'PHENOTYPE'
```

Comment:
I know it's just formatting, but should we be moving away from these? cc @matentzn @cmungall

Reply:
+1